server.php

```diff
@@ -65,9 +65,11 @@ function valet_support_xip_io($domain)
 $valetSitePath = null;
 
 foreach ($valetConfig['paths'] as $path) {
-    if (is_dir($path.'/'.$siteName)) {
-        $valetSitePath = $path.'/'.$siteName;
-
+    $domain = ($pos = strrpos($siteName, '.')) !== false
+                ? substr($siteName, $pos+1)
+                : null;
+    if (is_dir($path.'/'.$siteName) || is_dir($path.'/'.$domain)) {
```

Comment:
Hey @jralph can you describe how to reproduce this in more detail? I am running `dev-master` and able to hit non-subdomain sites without any issues.

Are you using `link` or `park`? Is it only for SSL sites or all sites?